generate_test.go

```diff
@@ -27,8 +27,9 @@ message test {
   required float f32 = 11;
   required double f64 = 12;
   required bytes bytes = 13;
-  required string string = 14;
-  required emb struct = 15;
+  required bytes array = 14;
+  required string string = 15;
+  required emb struct = 16;
```

Comment:
are you sure this is what would be generated as a `.proto` file? Look here: https://developers.google.com/protocol-buffers/docs/proto#nested